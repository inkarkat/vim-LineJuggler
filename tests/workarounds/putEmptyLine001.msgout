
3 more lines
"~/Unixhome/.vim/tests/LineJuggler/workarounds/putEmptyLine001.out" 
</.vim/tests/LineJuggler/workarounds/putEmptyLine001.out" [New] 5L, 53C written
