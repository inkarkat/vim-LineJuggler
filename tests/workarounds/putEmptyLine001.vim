" Test putting a list of lines that end with an empty line.

insert
first line
last line
.

call ingolines#PutWrapper(1, 'put', ['new 1', 'new 2; v empty line here', ''])

call vimtest#SaveOut()
call vimtest#Quit()
