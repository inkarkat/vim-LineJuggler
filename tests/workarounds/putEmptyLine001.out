first line
new 1
new 2; v empty line here

last line
